app/src/main/java/com/babylon/wallet/android/presentation/accessfactorsources/signatures/GetSignaturesViewModel.kt

```diff
@@ -128,19 +128,19 @@ class GetSignaturesViewModel @Inject constructor(
                 _state.update { state ->
                     state.copy(
                         showContentForFactorSource = State.ShowContentForFactorSource.Device,
-                        isRetryButtonEnabled = true // enable the Retry button, reason is at line 184
+                        isRetryButtonEnabled = true // enable the Retry button, reason is at line 185
```

Comment:
I would suggest removing line references as they change rapidly even if the methods that you are referencing not. You could reference the method.

Reply:
answered this [here](https://github.com/radixdlt/babylon-wallet-android/pull/1216#discussion_r1860084575).